tests/SeverityTest.php

```diff
@@ -62,4 +62,37 @@ public function testIsEqualTo(): void
         $this->assertTrue($severity1->isEqualTo($severity2));
         $this->assertFalse($severity1->isEqualTo($severity3));
     }
+
+    /**
+     * @dataProvider fromErrorDataProvider
+     */
+    public function testFromError(int $errorLevel, string $expectedSeverity): void
+    {
+        $this->assertSame($expectedSeverity, Severity::fromError($errorLevel)->__toString());
```

Comment:
Hiding them is a BC for sure. I did this just to avoid using `isEqualTo`, that would lead to depending on another test.

Reply:
>Hiding them is a BC for sure

Indeed, infact we should mark them as `@internal` and nothing more and in the next major version we can make them `private`

>I did this just to avoid using isEqualTo, that would lead to depending on another test

It's fine, we can compare against hard-coded `string`s instead of using the constants, this way we are also more confident that if we change the values we will see the BC